# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../kbd/devfs.patch
# Copyright (C) 2010 - 2011 The OpenSDE Project
# Copyright (C) 2004 - 2006 The T2 SDE Project
# Copyright (C) 1998 - 2004 Clifford Wolf
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- kbd-1.15.3/src/openvt.c.orig	2011-08-12 11:10:02.526038758 +0200
+++ kbd-1.15.3/src/openvt.c	2011-08-12 11:10:51.303377609 +0200
@@ -29,8 +29,8 @@
 
 // Where your VTs are hidden
 #ifdef __linux__
-#define VTNAME "/dev/tty%d"
-#define VTNAME2 "/dev/vc/%d"
+#define VTNAME "/dev/vc/%d"
+#define VTNAME2 "/dev/tty%d"
 #endif
 
 #ifndef VTNAME
